.buildkite/scripts/package.sh

```diff
@@ -20,6 +20,10 @@ if [[ ${PLATFORM_TYPE} == "arm" || ${PLATFORM_TYPE} == "aarch64" ]]; then
     PACKAGES="docker"
 fi
 
+if [[ ${FIPS} == "true" ]]; then
```

Comment:
Is this necessary?
If FIPS is set to true from the outside (as it seems the case here) it will have the same value for all the script called from here as well (it's not a variable that is local to this script)